cmd/vault_kv_helpers.go

```diff
@@ -0,0 +1,85 @@
+package cmd
```

Comment:
Original code licensed under MPL https://github.com/hashicorp/vault/blob/master/LICENSE. I believe we need to add a license header here: 
https://www.mozilla.org/en-US/MPL/headers/

Reply:
License added